SDK_versioned_docs/version-3.0.0/guides/auto-router/auto-router.md

```diff
@@ -0,0 +1,192 @@
+---
+id: quick-start
+sidebar_position: 1
+---
+
+# Quick Start
+You can use the auto router to fetch optimized trade routes for swapping on Uniswap. To use the auto router, you will create an `AlphaRouter` instance and use the method `route` to get quotes, gas information, and calldata for an optimized swap.
+
+In this quick start you will:
+
+- import necessary packages
+- initialize the `AlphaRouter`
+- set up the parameters for the `route` method
+- call `route` to retrieve calldata
+- submit a route transaction
+
+## Importing the Package
+To integrate with the auto router, you will use the [smart-order-router](https://www.npmjs.com/package/@uniswap/smart-order-router) package. The smart-order-router package allows you to integrate with the auto router feature through the `AlphaRouter` class. 
+
+Import `AlphaRouter` from the smart-order-router package to get started.
+
+```typescript
+import { AlphaRouter } from '@uniswap/smart-order-router'
+```
+
+The `AlphaRouter` class handles the routing logic. Instantiate the `AlphaRouter` to call the `route` method which returns all the swap calldata and gas information needed for submitting an optimal swap to the chain. 
+
+## Initializing the AlphaRouter
+
+To create an instance of the AlphaRouter, configure the  `chainId` and `provider` parameters.
+
+#### Parameters
+`chainId` [required]
+- The id of the chain you want to route swaps on. Ex) The chainId for Ethereum mainnet is `1`.
+
+`provider` [required]
+- A JSON RPC endpoint, like Infura.
+
+```typescript
+const router = new AlphaRouter({ chainId: 1, provider: web3Provider };
+```
+
+## Calling `route`
+
+Once you instantiate `AlphaRouter` call `route` with the following parameters:
+
+```typescript
+    {
+      amount: CurrencyAmount,
+      quoteCurrency: Currency,
+      swapType: TradeType,
+      swapConfig?: SwapConfig,
+      partialRoutingConfig?: Partial<AlphaRouterConfig> = {}
+    }
+```
+#### Parameters
+`amount` [required]
+- The amount to be swapped as a `CurrencyAmount`
+`quoteCurrency` [required]
+- The token or native currency you are swapping for as a `Currency`
```

Comment:
One note, if you are doing exactIn then quoteCurrency is your tokenOut, if exactOut then quoteCurrency is your tokenIn. May be worth calling out (I know this tripped me up when i started working on routing)